css/droplet.css

```
@@ -71,7 +71,7 @@
   position: absolute;
   top: 0; bottom: 0; width: 300px;
   max-width: 40%;
-  z-index: 257;
+  z-index:0;
```

Tiny style nit: missing space.
```suggestion
  z-index: 0;
```